test_fms/fms2_io/input.nml

```diff
@@ -1 +1 @@
-aljds;falkjsd
+blah;blahblah
```

Comment:
Is this meant to be here?

Reply:
Yes, unless others are opposed to having a bit of fun with a dummy file.

Reply:
I approve

Reply:
I understand the need for input.nml to exist because mpp_init reads it into the input_nml_file character array for use with namelist internal file reads (plus documenting it via the logfile).  But I think the cleanest solution is to have the test script "cat" a dummy namelist entry into the input.nml file:
cat input.nml < EOF
  &dummy /
EOF